src/libraries/System.Private.CoreLib/src/System/IO/InvalidDataException.cs

```diff
@@ -23,7 +23,7 @@ public InvalidDataException()
         /// <param name="message">The error message that explains the reason for the exception.</param>
         /// <remarks>This constructor initializes the <see cref="Exception.Message" /> property of the new instance to a system-supplied message that describes the error, such as "An invalid argument was specified." This message is localized based on the current system culture.</remarks>
```

Comment:
🌝: now behaviour slightly more matches remark.
🌚: it copied from [ArgumentException](https://learn.microsoft.com/en-us/dotnet/api/system.argumentexception.-ctor?view=net-8.0).